config/environments/development.rb

```diff
@@ -1,3 +1,6 @@
+require 'dotenv'
+Dotenv.load
```

Comment:
I've never need to do this to load dotenv before. It should load up fine without it.

Reply:
Good to know! The last time I worked with Ruby, `figaro` was still the gem everyone seemed to be using for this, so wasn't even sure that `dotenv` was the right one to go for! Thanks 🙏🏻